.github/workflows/changelog_reminder.yaml

```diff
@@ -0,0 +1,33 @@
+name: Verify changelog updated
+
+on: 
+  pull_request:
+    types:
+      - opened
+      - ready_for_review
+
+
+jobs:
+  check_changes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get changed files
+        id: getfile
+        run: |
+          echo "Files changed:"
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
+          echo "$CHANGED_FILES"
+          echo "CHANGED_FILES<<EOF" >> $GITHUB_ENV
+          echo "$CHANGED_FILES" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+          
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+           message: 'Thank you for the PR! The changelog has not been updated so here is a friendly reminder to check if you need at add an entry.'
```

Comment:
```suggestion
           message: 'Thank you for the PR! The changelog has not been updated, so here is a friendly reminder to check if you need to add an entry.'
```

Reply:
Wow I really need to learn to proofread, thanks for catching this!